### ISSUE TITLE
Plugin does not support authorization plugin model

### ISSUE THREAD

**Issue by psmaAaron** [koopjs/koop-output-vector-tiles#11]
As the title, this plugin does not do anything if there is an authorisation plugin loaded.

**Comment by rgwozdz** [comment]
Yes, that is correct.  Currently, the auth design requires each output to implement a certain spec.  We're working to change that in the coming months.
